src/signals/settings/roles/containers/RoleFormContainer/index.js

```diff
@@ -0,0 +1,127 @@
+import React, { Fragment, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { createStructuredSelector } from 'reselect';
+import { bindActionCreators } from 'redux';
+import { Row, Column } from '@datapunt/asc-ui';
+
+import PageHeader from 'signals/settings/components/PageHeader';
+import LoadingIndicator from 'shared/components/LoadingIndicator';
+import BackLink from 'components/BackLink';
+import FormAlert from 'components/FormAlert';
+
+import makeSelectRolesModel from 'models/roles/selectors';
+import { fetchRoles, fetchPermissions, patchRole } from 'models/roles/actions';
+
+import RoleForm from '../../components/RoleForm';
+
+export const RoleFormContainer = ({
+  roles: {
+    list,
+    permissions,
+    loading,
+    loadingPermissions,
+    responseSuccess,
+    responseError,
+  },
+  onFetchRoles,
+  onFetchPermissions,
+  onPatchRole,
+}) => {
+  useEffect(() => {
+    onFetchRoles();
+    onFetchPermissions();
+  }, []);
+
+  const { roleId } = useParams();
+  const role = list.find(item => item.id === roleId * 1);
+
+  return (
+    <Fragment>
+      <PageHeader
+        title="Rol instellingen"
+        BackLink={
+          <BackLink to="/instellingen/rollen">
+            Terug naar overzicht
+          </BackLink>
+        }
+      />
+      <Row>
+        <Column span={12}>
+          {responseSuccess && <FormAlert
+            data-testid="roleFormSuccess"
+            isNotification
+            title="Gegevens opgeslagen"
+          />}
+          {responseError &&
```

Comment:
There is a list of error messages, mapped by status code, available from `shared/services/api/api`. You should use that instead. The text `Er is iets mis gegaan bij het opslaan` is too generic to show to the user.